src/modelinstance.cpp

```diff
@@ -668,6 +668,7 @@ Status ModelInstance::loadOVCompiledModel(const ModelConfig& config) {
     plugin_config_t pluginConfig = prepareDefaultPluginConfig(config);
     try {
         loadCompiledModelPtr(pluginConfig);
+        SET_IF_ENABLED(getMetricReporter().streams, getNumOfStreams());
```

Comment:
```suggestion
        SET_IF_ENABLED(this->getMetricReporter().streams, getNumOfStreams());
```
it can be without "this->" for me as well but be consistent.

Reply:
Could we log this value as well? Checking streams against nireq is usefull to have in logs

Reply:
Added log